### PR TITLE
Redraw on resize

### PR DESCRIPTION
**Pull request by ym1234**

### CODE REVIEW

**Comment by ym1234** [comment]
I think we should also use sigaction instead of signal since it is deprecated.

File: filet.c [hunk 1]
```diff
@@ -40,6 +40,7 @@ struct direlement {
 static struct termios g_old_termios;
 static int g_row;
 static int g_col;
+static bool needs_redraw;
```
Comment (buffet) [inline filet.c]: Can you rename this to `g_needs_redraw`, to indicate it's a global variable, please.

File: filet.c [hunk 2]
```diff
@@ -431,12 +435,27 @@ main(int argc, char **argv)
...
+            if (y > scroll_size) {
+                y = scroll_size;
+            } else if (empty_space > 0) {
+                y = n >= scroll_size ? y + empty_space + 1 : sel;
+            }
+            redraw(ents, user_and_hostname, path, n, sel, sel - y);
 
-            redraw(ents, user_and_hostname, path, n, sel, 0);
+            // Move cursor to selection
+            printf("\033[%zuH", y + 3);
```
Comment (buffet) [inline filet.c]: What's the reason you took this out of the redraw function?

Comment (ym1234) [reply filet.c]: The redraw function always sets this to the beginning of the scrolling region, while I need it to be at the current selection, and I didn't remove it from redraw (although maybe I should?)

Comment (buffet) [reply filet.c]: Yes, you should. Can you lowercase the `M` to match with the rest of the comments, while you're at it?

**Pull request merged**